apache/superset
# feat: Add feature flag to enable/disable creation of 'examples' database.

### SUMMARY
Add feature flag `CREATE_EXAMPLES_DATABASE` to enable/disable the creation of 'examples' database during datasets import.

### TEST PLAN
- `CREATE_EXAMPLES_DATABASE=False`:
  - import datases;
  - Verify that 'examples' database was not  created.
- `CREATE_EXAMPLES_DATABASE=True`:
  - import datases;
  - Verify that 'examples' database was created.

### ADDITIONAL INFORMATION
Fixes #14050
